src/biotite/database/uniprot/query.py

```diff
@@ -189,7 +190,7 @@ def search(query, number=10):
     params = {
         'query': str(query),
         'format': 'list',
-        'limit': str(number)
+        'size': str(number)
```

Comment:
A suggestion for this PR or a future change: The new REST API allows pagination similar to the RCSB PDB. Hence, we could substitute the `number` parameter with `range` analogous to [database.rcsb.search](https://www.biotite-python.org/apidoc/biotite.database.rcsb.search.html). In the default case this would return all entries. Unfortunately, this could only be done in multiple requests as shown in https://www.uniprot.org/help/api_queries.

Reply:
I will keep this in mind for a future change.